### PR TITLE
Adding coverage for Content Hosts Subscription status widget in Dashboard (BZ1405524)

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/ui/test_dashboard.py -m test_positive_content_host_subscription_status
.
----------------------------------------------------------------------
Ran 1 test in 421.973s

OK
```

### CODE REVIEW

File: tests/foreman/ui/test_dashboard.py [hunk 1]
```diff
@@ -578,7 +587,6 @@ def test_positive_sync_overview_widget(self):
                 'Syncing Complete.'
             )
 
-    @stubbed()
     @tier2
     def test_positive_content_host_subscription_status(self):
```
Comment (ldjebran) [inline tests/foreman/ui/test_dashboard.py]: perhaps we need to run in one thread, what do you think

File: tests/foreman/ui/test_dashboard.py [hunk 2]
```diff
@@ -594,16 +602,51 @@ def test_positive_content_host_subscription_status(self):
...
+        :Assert: The widget is updated with all details for Valid, Invalid
+            and Partial Subscriptions
 
         :CaseLevel: Integration
         """
+        org = entities.Organization().create()
+        env = entities.LifecycleEnvironment(
+            organization=org).create()
+        content_view = entities.ContentView(
+            organization=org).create()
```
Comment (abalakh) [inline tests/foreman/ui/test_dashboard.py]: nitpick: fits in 1 line

File: tests/foreman/ui/test_dashboard.py [hunk 3]
```diff
@@ -594,16 +602,51 @@ def test_positive_content_host_subscription_status(self):
...
 
-        :caseautomation: notautomated
+        :Assert: The widget is updated with all details for Valid, Invalid
+            and Partial Subscriptions
 
         :CaseLevel: Integration
         """
+        org = entities.Organization().create()
+        env = entities.LifecycleEnvironment(
+            organization=org).create()
```
Comment (abalakh) [inline tests/foreman/ui/test_dashboard.py]: nitpick: fits in 1 line

**Comment by oshtaier** [comment]
Don't need to have that case in one thread as it is completely independent from any other test case, but added requirements for configuration. Thanks

**Comment by abalakh** [comment]
@oshtaier not sure if i got your point. In what way is it independent? Test uploads manifest, which may cause race conditions if any other thread will try to upload a manifest to any other org.

**Comment by oshtaier** [comment]
@abalakh Thank you for your extremely valuable answer. Please, merge the PR if you don't have any other suggestions

**Pull request merged**